Source/WebCore/mathml/MathMLPresentationElement.cpp

```diff
@@ -182,7 +182,6 @@ bool MathMLPresentationElement::isFlowContent(const Node& node)
         || htmlElement.hasTagName(HTMLNames::pTag)
         || htmlElement.hasTagName(HTMLNames::preTag)
         || htmlElement.hasTagName(HTMLNames::sectionTag)
-        || (htmlElement.hasTagName(HTMLNames::styleTag) && htmlElement.hasAttribute(HTMLNames::scopedAttr))
```

Comment:
Aside: I filed https://github.com/w3c/mathml-core/issues/195 on this function as this seems rather suspect.